### ISSUE TITLE
start time of lfp data

### ISSUE THREAD

**Issue by bendichter** [datajoint/element-array-ephys#51]
For LFP, I am trying to handle if the probes were started at different times. I see a sampling rate, but no starting time. Contrast this with `ephys.EphysRecording` which has a `recording_datetime` field. For LFP, I see `lfp_time_stamps`, but the ones I can find start at 0, and it’s unclear whether this is time wrt the start of the LFP recording or the start of the session. `recording_datetime`  is the same for both probes in the example data, so that is consistent with the lfp time stamps being aligned, but it is unclear to me how it would be reflected in the data if the recordings started at different times. Would the `lfp_time_stamps` reflect this by having one of them not start at 0? Or would I be expected to infer the difference in start time by looking up the `recording_datetime`  from the same probe and session in the `ephys.EphysRecording`  table?

**Comment by kabilar** [comment]
Hi @bendichter, 

For a given probe and session, the `recording_datetime` in `ephys.EphysRecording` is when the `*.ap.meta` file was created.  See [ephys.EphysRecording section](https://github.com/ttngu207/element-array-ephys/blob/f9a4754355fe556c6a7ae28f068556568a9966a1/element_array_ephys/ephys_no_curation.py#L232-L276) and [spikeglx.SpikeGLX section](https://github.com/ttngu207/element-array-ephys/blob/f9a4754355fe556c6a7ae28f068556568a9966a1/element_array_ephys/readers/spikeglx.py#L192-L194).

The `lfp_time_stamps` are with regard to the start of the LFP recording, and would thus always start at 0.  [See lfp_time_stamps calculation in ephys.LFP](https://github.com/ttngu207/element-array-ephys/blob/f9a4754355fe556c6a7ae28f068556568a9966a1/element_array_ephys/ephys_no_curation.py#L369-L370).

@ttngu207, can you please confirm?

**Comment by bendichter** [comment]
OK so I should look up the corresponding `ephys.EphysRecording::recording_datetime` for the start of LFP signal?

**Comment by kabilar** [comment]
Yes, I believe so.  See `lfp_time_stamps` [attribute comment](https://github.com/ttngu207/element-array-ephys/blob/f9a4754355fe556c6a7ae28f068556568a9966a1/element_array_ephys/ephys_no_curation.py#L335).

My notes above were for SpikeGLX data.  I am confirming the same is the case for OpenEphys data.
